languages/js/src/Polar.test.ts

```diff
@@ -503,6 +504,24 @@ describe('#registerConstant', () => {
   });
 });
 
+describe('unifying promises', () => {
+  test('fails if both promises are the same object', () => {
```

Comment:
Is this testing the currently not-supported behaviour?

Reply:
Yeah, these tests will be updated when we support operations on resolved Promises.